distributed/comm/tests/test_ucx.py

```diff
@@ -56,8 +56,7 @@ def test_ucx_specific():
 
         async def handle_comm(comm):
             # XXX: failures here don't fail the build yet
-            import pdb; pdb.set_trace()
-            msg = yield comm.read()
+            msg = await comm.read()
```

Comment:
I'm guessing this isn't the only time I make this mistake :) Look for strange GeneratorExits.